src/App.js

```diff
@@ -8,11 +8,38 @@ import './App.scss';
 // Notifications
 import { NotificationsPortal } from '@redhat-cloud-services/frontend-components-notifications';
 
+export const PermissionContext = createContext();
+
 class App extends Component {
 
+    constructor() {
+        super();
+        this.state = {
+            readPermission: undefined,
+            writePermission: undefined,
+            executePermission: undefined
+        };
+    }
+
+    handlePermissionUpdate = (hasRead, hasWrite, hasExecute) => this.setState({
+        readPermission: hasRead,
+        writePermission: hasWrite,
+        executePermission: hasExecute
+    });
+
     componentDidMount () {
         insights.chrome.init();
         insights.chrome.identifyApp('remediations');
+        window.insights.chrome.getUserPermissions().then(
```

Comment:
This fetches all permissions which is wasteful. It should be enough to pass "remediations" to the function

Reply:
However, even when I change the code to pass the app name it still does not seem to be making the correct REST request :-/

Reply:
This is currently calling the chrome function. I don't believe we have a way to filter out a certain app at this time.

Reply:
@ryelo isn't it what this first param does? https://github.com/RedHatInsights/insights-chrome/blob/master/src/js/entry.js#L125

Reply:
Or rather supposed to do? Like I said when testing it did not seem to work

Reply:
It is exactly what @jharting is refering to, it is strange though because I tested it and it does exactly what was expected. It queries the RBAC server with applications parameter in query URL

Reply:
This is working for me! I'll update accordingly